libethcore/Miner.h

```diff
@@ -81,6 +81,31 @@ inline std::ostream& operator<<(std::ostream& _out, WorkingProgress _p)
 	return _out;
 }
 
+struct FarmWorkingProgress
```

Comment:
@chfast  You mean something like this : https://gist.github.com/evilny0/19f2172a7f45a3b00bfd00ec0e8290c2 ?